lib/AppInfo/Application.php

```diff
@@ -61,6 +63,7 @@ public function __construct() {
 	public function register(IRegistrationContext $context): void {
 		/** Register $principalBackend for the DAV collection */
 		$context->registerServiceAlias('principalBackend', Principal::class);
+		$context->registerEventListener(NodeDeletedEvent::class, MoveToTrashListener::class);
```

Comment:
Maybe this instead?
Trashbin can be disabled :)
```suggestion
		$context->registerEventListener(NodeDeletedEvent::class, NodeDeletedListener::class);
```

Reply:
`MoveToTrashListener` is the name of our listener class. Not sure I get your point :thinking:

Reply:
Just talking about the name choice :p
A very critical problem obviously :see_no_evil: /s